ethereum/package.json

```diff
@@ -11,7 +11,7 @@
     "postinstall": "cd node_modules/solidity-parser-antlr && yarn install && yarn build",
     "compile": "npx saddle compile",
     "test": "npx saddle compile && npx saddle test",
-    "coverage": "npx saddle compile --trace && npx saddle coverage && npx istanbul report --root=./coverage lcov json",
+    "coverage": "BUILD_FILE=.build/contracts-trace.json npx saddle compile --trace && BUILD_FILE=.build/contracts-trace.json npx saddle coverage && npx istanbul report --root=./coverage lcov json",
```

Comment:
FYI, you probably don't need the second `BUILD_FILE` env var here.

Reply:
No, unfortunately it needs it